### DISCUSSION TITLE
pixel spacing issue

### DISCUSSION THREAD

**Discussion by davidvillaloboschaves** [MorphoSource/Community#109] [General]
Dear community! 

I a little bit confused if I can get pixel spacing information from the log document created by the CT scanner? As it is such an important information for uploading data to Morphosource, I want to make sure. 

I would appreciate any advice. Muchas gracias!!!

Note: I haven't tried getting the data from ImageJ as I am trying to save some time. In case it helps, there is one txt file with the log information attached to this question. Thanks

[Anoura_geoffroyi_UWBM61987_121421_.log](https://github.com/user-attachments/files/16132029/Anoura_geoffroyi_UWBM61987_121421_.log)

**Comment by JulieWinchester** [comment]
Hi David! I'm not an expert in Skyscan log files, but I believe the information you're looking for here is on line 31:

```
Image Pixel Size (um)=   11.86
```

Probably the X pixel spacing, Y pixel spacing, and Z pixel spacing fields should all be 11.86, with the unit field set to micron/micrometer. This is of course assuming that this scan has uniform pixel spacing across all dimensions.

**Reply by davidvillaloboschaves** [reply]
Thank you so much!
